helpdesk_mgmtsystem_nonconformity/models/helpdesk_ticket_stage.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HelpdeskTicketStage(models.Model):
+    _inherit = "helpdesk.ticket.stage"
+
+    nonconformity_stage_id = fields.Many2one(
+        comodel_name="mgmtsystem.nonconformity.stage", string="Nonconformity Stage",
+    )
+    _sql_constraints = [
```

Comment:
Remove this constraint, as you can have several stages that leads to the same non conformity stage.